src/Frontend/components/app/layout/AppMenu.js

```diff
@@ -33,4 +30,17 @@ AppMenu.propTypes = {
   config: React.PropTypes.arrayOf(React.PropTypes.object)
 };
 
-export default AppMenu;
+const mapStateToProps = state => {
+  return ({
```

Comment:
I think we forgot to revert this yesterday after debugging. Should be `const mapStateToProps = state => ({ ... })`